### DISCUSSION TITLE
HELP! Updating Meeting Removes Location, Address & Region!

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#792] [Q&A] [answered]
I updated a meeting (just added some meeting notes) and the location name, address & region disappeared. No matter how many times I update, it reverts back to removing the location name, address & region.

This is with version 3.14.2

And it doesn't matter whether I am using API Gateway Beta or Legacy.

I also added another meeting, and it doesn't retain the name, address & region after publishing.

Please tell me how I can rollback to the previous plugin version which did not have this issue.

**Comment by kiyote33** [comment, marked as answer]
Hi @Grateful-Viv, I'm sorry to hear you are having this issue. I did some quick testing in my sandbox, doing meeting note updates using the 3.14.2 version. I was not able to replicate the issue. If you could give me specific details on your situation, I can try again to replicate.  

As for rolling back, one way you can download the 3.14.1 version is to go to our [wordpress plugin page](https://wordpress.org/plugins/12-step-meeting-list/advanced/).  Previous versions (at the bottom of the page) can be selected and downloaded.  Another way is to install the wp rollback plugin and it facilitates rolling back to previous versions. 

Hope this helps...

**Reply by Grateful-Viv** [reply]
Thanks for the info on where to get previous versions.

As suddenly as this issue appeared, which also happened then on my staging site, it resolved itself. All in all - it was about 1/2 hour of panic. I have no idea why this happened. In the 3+ years we have been using the plugin, this was never an issue that occurred.
